### DISCUSSION TITLE
Understrap’s Premium Child Themes Demo Content

### DISCUSSION THREAD

**Discussion by Matteo182** [understrap/understrap-child#333] [General]
That is, I downloaded the new premium child themes, but I can't find the demo content like in the old ones, don't you have a file to download the demo contet so that it is ready as it was before? 

thanks

**Comment by bacoords** [comment]
Hi @Matteo182 

Instead of coming with demo content, the latest versions of the premium child themes come with Block Patterns that can be used on any page/post using the WordPress block editor. You can use these block patterns to create the same home/landing page designs that were in the previous version of the child themes- or mix and match and use them site wide.

While we finish up some documentation for these child themes, here's some basic instructions on what Block Patterns are and how to insert them: https://wordpress.org/support/article/block-pattern/

**Reply by Matteo182** [reply]
thanks for the answer, in fact it is much more useful.
